interfaces/functions.yaml

```diff
@@ -550,6 +550,11 @@ igraph_distances_floyd_warshall:
         GRAPH graph, OUT MATRIX res, EDGEWEIGHTS weights=NULL, NEIMODE mode=OUT
     DEPS: weights ON graph
 
+igraph_voronoi:
+    PARAMS: |-
+        GRAPH graph, OUT VECTOR_INT membership, OUT VECTOR distances,
+        VECTOR_INT generators, VECTOR weights, NEIMODE mode=OUT, VORONOI_TIEBREAKER tiebreaker=RANDOM
+
```

Comment:
Should you not have some DEPS here?

Reply:
Thanks, added the dependency for weights.